extension/inet/include/html_charref.hpp

```diff
@@ -0,0 +1,4552 @@
+		mapped_strings[string_t(owned_strings[123])] = 10583;
+		mapped_strings[string_t(owned_strings[124])] = 8868;
+		mapped_strings[string_t(owned_strings[125])] = 861;
+		mapped_strings[string_t(owned_strings[126])] = 8659;
+		mapped_strings[string_t(owned_strings[127])] = 119967;
+		mapped_strings[string_t(owned_strings[128])] = 27;
+		mapped_strings[string_t(owned_strings[129])] = 330;
+		mapped_strings[string_t(owned_strings[130])] = 208;
+		mapped_strings[string_t(owned_strings[131])] = 20;
+		mapped_strings[string_t(owned_strings[132])] = 201;
+		mapped_strings[string_t(owned_strings[133])] = 201;
+		mapped_strings[string_t(owned_strings[134])] = 28;
+		mapped_strings[string_t(owned_strings[135])] = 202;
+		mapped_strings[string_t(owned_strings[136])] = 202;
+		mapped_strings[string_t(owned_strings[137])] = 106;
+		mapped_strings[string_t(owned_strings[138])] = 278;
+		mapped_strings[string_t(owned_strings[139])] = 120072;
+		mapped_strings[string_t(owned_strings[140])] = 20;
+		mapped_strings[string_t(owned_strings[141])] = 200;
+		mapped_strings[string_t(owned_strings[142])] = 8712;
+		mapped_strings[string_t(owned_strings[143])] = 27;
+		mapped_strings[string_t(owned_strings[144])] = 9723;
+		mapped_strings[string_t(owned_strings[145])] = 9643;
+		mapped_strings[string_t(owned_strings[146])] = 28;
+		mapped_strings[string_t(owned_strings[147])] = 120124;
+		mapped_strings[string_t(owned_strings[148])] = 917;
+		mapped_strings[string_t(owned_strings[149])] = 1086;
+		mapped_strings[string_t(owned_strings[150])] = 8770;
+		mapped_strings[string_t(owned_strings[151])] = 8652;
+		mapped_strings[string_t(owned_strings[152])] = 849;
+		mapped_strings[string_t(owned_strings[153])] = 10867;
+		mapped_strings[string_t(owned_strings[154])] = 919;
+		mapped_strings[string_t(owned_strings[155])] = 20;
+		mapped_strings[string_t(owned_strings[156])] = 203;
+		mapped_strings[string_t(owned_strings[157])] = 8707;
+		mapped_strings[string_t(owned_strings[158])] = 851;
+		mapped_strings[string_t(owned_strings[159])] = 1060;
+		mapped_strings[string_t(owned_strings[160])] = 120073;
+		mapped_strings[string_t(owned_strings[161])] = 972;
+		mapped_strings[string_t(owned_strings[162])] = 9642;
+		mapped_strings[string_t(owned_strings[163])] = 120125;
+		mapped_strings[string_t(owned_strings[164])] = 870;
+		mapped_strings[string_t(owned_strings[165])] = 8497;
+		mapped_strings[string_t(owned_strings[166])] = 8497;
+		mapped_strings[string_t(owned_strings[167])] = 102;
+		mapped_strings[string_t(owned_strings[168])] = 62;
+		mapped_strings[string_t(owned_strings[169])] = 62;
+		mapped_strings[string_t(owned_strings[170])] = 91;
+		mapped_strings[string_t(owned_strings[171])] = 988;
+		mapped_strings[string_t(owned_strings[172])] = 286;
+		mapped_strings[string_t(owned_strings[173])] = 29;
+		mapped_strings[string_t(owned_strings[174])] = 284;
+		mapped_strings[string_t(owned_strings[175])] = 1043;
+		mapped_strings[string_t(owned_strings[176])] = 28;
```

Comment:
To reiterate the point made below:
`"&Gdot;": { "codepoints": [288], "characters": "\u0120" }`